test/e2e/framework/device_plugin_util.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+import (
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	. "github.com/onsi/gomega"
+)
+
+const (
+	SampleResourceName = "fake.com/resource"
+	// TODO: Parametrize it by making it a feature in TestFramework.
+	SampleDevicePluginDSYAML = "https://gist.githubusercontent.com/vikaschoudhary16/75a4b609112374253a4ec98b57251b83/raw/0560df2a7ea69bfc4e1dd6dda0aa1f7008c7fcab/sample-device-plugin-daemonset.yaml"
```

Comment:
Can we reference the yaml in `test/e2e/testing-manifests/sample-device-plugin.yaml` instead of this url?